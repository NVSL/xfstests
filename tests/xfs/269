#! /bin/bash
# FS QA Test 269
#
# what am I here for?
#
#-----------------------------------------------------------------------
# Copyright (c) 2016 YOUR NAME HERE.  All Rights Reserved.
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU General Public License as
# published by the Free Software Foundation.
#
# This program is distributed in the hope that it would be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program; if not, write the Free Software Foundation,
# Inc.,  51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
#-----------------------------------------------------------------------
#

seq=`basename $0`
seqres=$RESULT_DIR/$seq
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1	# failure is the default!
trap "_cleanup; exit \$status" 0 1 2 3 15

_cleanup()
{
	cd /
	rm -f $tmp.*
}

# get standard environment, filters and checks
. ./common/rc
. ./common/filter

# remove previous $seqres.full before test
rm -f $seqres.full

# real QA test starts here

# Modify as appropriate.
_supported_fs generic
_supported_os IRIX Linux
_require_test

index=10
setup_cmd="_nova_create_files"
modify_state="_nova_modify_inode_state"
capture_state="_nova_capture_inode_state"
cleanup_cmd="_nova_delete_files"

_nova_test_snapshots $setup_cmd $modify_state $capture_state $cleanup_cmd $index


# if error
# exit

# optional stuff if your test has verbose output to help resolve problems
#echo
#echo "If failure, check $seqres.full (this) and $seqres.full.ok (reference)"

# success, all done

status=0
exit
