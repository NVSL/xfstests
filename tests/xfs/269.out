QA output created by 269
